format-definitions/parsec-corim-tpm.md

```diff
@@ -0,0 +1,119 @@
+# Parsec TPM endorsement
```

Comment:
We also need to supply the template for the outer CoRIM here.

Something like:
```json
{
  "corim-id": "ffffffff-ffff-ffff-ffff-ffffffffffff",
  "profiles": [
    "http://github.com/parallaxsecond/tpm/0.0.1"
  ]
}
```